manuscript/teaching-reproducibility.Rmd

```diff
@@ -290,9 +290,30 @@ This example highlights the enormous cost
 associated with not adhering to reproducible practices
 and having workflows that are opaque and hard to review.
 
+As illustrated in the case we have highlighted above,
+reproducibility errors are not isolated to novices performing irrelevant data analyses.
```

Comment:
"irrelevant" is not great here - I think it minimizes the meaning of data analyses performed by novices. I suggest we remove it. 
```suggestion
reproducibility errors are not isolated to novices performing data analyses.
```